### ISSUE TITLE
Any order operator

### ISSUE THREAD

**Issue by mynomoto** [juxt/iota#6]
Would be of interest to have something like `:=~` for comparing vectors where the order doesn't matter? So `[:a :b :c] :=~ [:b :a :c]`?

**Comment by danielcompton** [comment]
As an alternative, you could accomplish this by calling `set` on the result, 

For example: `[:path set] := #{:b :a :c}`.

**Comment by mynomoto** [comment]
Yeah, I got it working with two assertions `:>` and `:<`, your solution is more elegant. But in the end I moved to testit. The syntax was more formatter friendly.
